### ISSUE TITLE
Allow str + Enum for WPS600 Forbid subclassing lowercase builtins.

### ISSUE THREAD

**Issue by gotofritz** [wemake-services/wemake-python-styleguide#2506]
# Rule request

"WPS600 Forbid subclassing lowercase builtins" should allow the following

```python
class PizzaToppings(str, Enum):
    MARGHERITA = "MARGHERITA"
    MARINARA = "MARINARA"
```

## Thesis

Enums are a bit akwkard in Python. The `class PizzaToppings(str, Enum):...` construct allows to create what is basically a string equivalent of IntEnum, so that one can do

```python
if current_topping == PizzaToppings.MARINARA:
    ...
```
 
## Reasoning

It seems beneficial, farily widely used, and has no negative side effects I can think of. No reason to ban it. 

Thanks.

**Comment by sobolevn** [comment]
Yes, this is actually a good exception for this rule.
python3.11 has `StrEnum` to help.

**Comment by lensvol** [comment]
Can I take this?

**Comment by bronzeson** [comment]
We're going to ignore this warning specifically for this case. Is there any possible problem that could arise?
